### ISSUE TITLE
gnt-instance modify -t drbd doesn't use instance allocator

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#851]
Originally reported of Google Code with ID 795.

```
When converting an instance from plain to drbd using "gnt-instance modify -t drbd", the -n parameter to specify the secondary node is mandatory.

I would have expected that the -n parameter could be omitted and would use the instance allocator (for example like "gnt-instance add -t drbd", or "gnt-instance replace-disks --iallocator .")

System info:

# gnt-cluster --version
gnt-cluster (ganeti v2.10.2) 2.10.2
# gnt-cluster version
Software version: 2.10.2
Internode protocol: 2100000
Configuration format: 2100000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version v2.10.2
# hspace --version
hspace (ganeti) version v2.10.2
compiled with ghc 7.4
running on linux x86_64

Host OS: Debian Wheezy

```
Originally added on 2014-04-10 12:13:21 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-05-20 12:34:12 +0000 UTC.

Added Labels: Type-Defect Priority-Medium
Changed State: Accepted
Added to Milestone: Release2.13

**Comment by Ganeti-Issues-Migrator** [comment]
```
Implemented and will be part of the 2.16 release.
```
Originally added on 2015-07-22 13:41:30 +0000 UTC.

Added Labels: Type-Enhancement
Removed Labels: Type-Defect
Changed State: Fixed
Added to Milestone: Release2.16
Removed from Milestone: Release2.13
